vgen/src/test/java/io/gapi/vgen/testdata/go_library.baseline

```diff
@@ -191,14 +189,14 @@ func (book Book) Path() (string, error) {
 
 // CreateShelf creates a shelf, and returns the new Shelf.
 func (client *Client) CreateShelf(ctx context.Context, req *google_example_library_v1.CreateShelfRequest, opts ...gax.CallOption) (*google_example_library_v1.Shelf, error) {
-	callOpts := append(nonIdempotentRetryCodes(), defaultRetryParams(), client.CallOptions["CreateShelf"]...)
-	callOpts = append(callOpts, opts...)
+	defaultCallOpts := append(nonIdempotentRetryCodes(), defaultRetryParams()...)
+	callOpts := append(client.callOptions["CreateShelf"], opts...)
```

Comment:
This reminds me of a discussion in the internal code review. append() has an optimization that if the latter part (client.callOptions["CreateShelf"] in this case) has enough space, append() actually updates it rather than creating a new slice, thus this operation is not very safe sometimes. You may want to make a copy instead.